supplementary/devices/index.rst

```diff
@@ -32,6 +32,7 @@ All of these should only be document in the exact same way here and **only an ov
    pme
    rolex
    T00F
+   trlte
```

Comment:
the list is almost alphabetical. if you move it down one line, we're at least not making it worse

Reply:
alphabetized the list